imagine/main.go

```diff
@@ -230,10 +230,12 @@ func main() {
 	}
 
 	if conf.Generate {
+		genStart := time.Now()
```

Comment:
doesn't ApplyWorkloads already have a time thing for this? i guess it might be missing. If we do add this, though, it should possibly be contingent on conf.Time?